CHANGES.md

```diff
@@ -1,13 +1,15 @@
 unreleased
 -------------------
 
+- Add `-unused-code-warnings` command-line flag. (#444, @ceastlund)
```

Comment:
```suggestion
- Driver: Add `-unused-code-warnings` command-line flag. (#444, @ceastlund)
```

Talking about "command-line" makes it quite clear that the change is about the driver, not the library, but it's still helpful to be explicit.